stock_move_value_report/report/stock_move_line_value_report.xml

```diff
@@ -171,33 +173,39 @@
                     <t t-if="ml_from_scraps">
                         <t
                             t-set="scraps"
-                            t-value="move_lines_ids.mapped('move_id.scrap_ids').filtered(lambda x: x.move_id in ml_from_scraps.mapped('move_id'))"
+                            t-value="move_lines_ids.mapped('move_id.scrap_id').filtered(lambda x: x in ml_from_scraps.mapped('move_id.scrap_id'))"
```

Comment:
why do you map on `scrap_id` here:
`.filtered(lambda x: x in ml_from_scraps.mapped('move_id.scrap_id'))`?